tools/reviewer/local_server/service/tests/TestTool.java

```diff
@@ -16,27 +16,39 @@
 
 package com.google.startupos.tools.reviewer.localserver.service.tests;
 
+import com.google.startupos.common.repo.Protos.File;
 import com.google.startupos.tools.reviewer.localserver.service.CodeReviewServiceGrpc;
-import com.google.startupos.tools.reviewer.localserver.service.Protos.DiffRequest;
+import com.google.startupos.tools.reviewer.localserver.service.Protos.CreateDiffRequest;
+import com.google.startupos.tools.reviewer.localserver.service.Protos.Diff;
 import com.google.startupos.tools.reviewer.localserver.service.Protos.DiffFilesRequest;
 import com.google.startupos.tools.reviewer.localserver.service.Protos.DiffFilesResponse;
+import com.google.startupos.tools.reviewer.localserver.service.Protos.DiffRequest;
+import com.google.startupos.tools.reviewer.localserver.service.Protos.FileRequest;
+import com.google.startupos.tools.reviewer.localserver.service.Protos.TextDiffRequest;
+import com.google.startupos.tools.reviewer.localserver.service.Protos.TextDiffResponse;
 import io.grpc.ManagedChannel;
 import io.grpc.ManagedChannelBuilder;
 import io.grpc.StatusRuntimeException;
 import java.util.concurrent.TimeUnit;
-import com.google.startupos.tools.reviewer.localserver.service.Protos.CreateDiffRequest;
-import com.google.startupos.tools.reviewer.localserver.service.Protos.Diff;
-import com.google.startupos.common.repo.Protos.File;
-import com.google.startupos.tools.reviewer.localserver.service.Protos.FileRequest;
-import com.google.startupos.tools.reviewer.localserver.service.Protos.TextDiffRequest;
-import com.google.startupos.tools.reviewer.localserver.service.Protos.TextDiffResponse;
 
 /* Test tool for CodeReviewService.
  *
  * Alternatively, to run grpc_polyglot (curl-like tool for gRPC), use:
- * cat tools/reviewer/local_server/service/tests/get_diff_files_request.json | bazel run //tools:grpc_polyglot -- --command=call --endpoint=localhost:8001 --full_method=com.google.startupos.tools.reviewer.localserver.service.CodeReviewService/getDiffFiles
- * cat tools/reviewer/local_server/service/tests/get_text_diff_request.json | bazel run //tools:grpc_polyglot -- --command=call --endpoint=localhost:8001 --full_method=com.google.startupos.tools.reviewer.localserver.service.CodeReviewService/getTextDiff
- * cat tools/reviewer/local_server/service/tests/get_diff_request.json | bazel run //tools:grpc_polyglot -- --command=call --endpoint=localhost:8001 --full_method=com.google.startupos.tools.reviewer.localserver.service.CodeReviewService/getDiff
+ * cat tools/reviewer/local_server/service/tests/get_diff_files_request.json | \
+ *    CRS="com.google.startupos.tools.reviewer.localserver.service.CodeReviewService" \
```

Comment:
Why did we add `CRS`?

Reply:
@oferb lines were too long even when splitting arguments 🙂